examples/chat/store/getters.js

```diff
@@ -12,3 +12,15 @@ export const currentMessages = state => {
     ? thread.messages.map(id => state.messages[id])
     : []
 }
+
+export const unreadCount = state => {
+  const { threads } = state
+  return Object.keys(threads).reduce((count, id) => {
+    return threads[id].lastMessage.isRead ? count : count + 1
+  }, 0)
+}
+
+export const sortedMessages = state => {
+  const messages = currentMessages(state)
```

Comment:
I think using resolved getter is better rather than directly use getter function.

```js
export const sortedMessages = (state, getters) => {
  const messages = getters.currentMessages
```